lib/store/src/autoTitle.test.ts

```diff
@@ -40,6 +40,15 @@ describe('autoTitle', () => {
       ).toMatchInlineSnapshot(`Atoms/To/File`);
     });
 
+    it('match with trailing duplicate', () => {
+      expect(
+        auto(
+          './path/to/button/button.stories.js',
+          normalizeStoriesEntry({ directory: './path' }, options)
+        )
+      ).toMatchInlineSnapshot(`To/Button`);
+    });
+
```

Comment:
What's the outcome of a test with this?
```js
auto(
  './path/to/button/index.stories.js',
  normalizeStoriesEntry({ directory: './path' }, options)
)
```

Reply:
`To/Button/Index` -- do you think we should remove `Index`?

Reply:
I'd say so, normally `index` is ommited when importing e.g. 
```js
import { Button } from './components/Button'
```

so personally I'd expect a similar behavior. @tmeasday @ndelangen WDYT?

Reply:
Yeah, `"Index"` is meaningless to everyone, we should ignore it.

Reply:
Done. Thanks gang! 🙏

Reply:
Do we also cope with just `stories.js`? I feel like that filename would make sense for a folder.

Reply:
There was an issue in the eslint-plugin-storybook repo about supporting `stories.js` files. We never suggest that though, and I believe our internal defaults are all about `*.stories.*`. Do you think we should support `stories.*` ?

Reply:
I've seen it before, but I'm OK with saying it's not something we recommend.